cmd/cc/client.go

```diff
@@ -120,7 +120,13 @@ func main() {
 		os.Exit(1)
 	}
 
-	appConfig := utils.LoadClientConfig()
+	appConfig, err := utils.LoadClientConfig()
+
+	if err != nil {
+		setupLog.Error(err, "failed to load client configuration")
+		os.Exit(1)
```

Comment:
Is there a reason for not using `log.Fatalf()` like in `cmd/api/api.go` at line 45?

>  appConfig := utils.LoadApiConfig()
> 	appConfig, err := utils.LoadApiConfig()
> 	if err != nil {
> 		log.Fatalf("Cannot load the api configuration: '%v'", err.Error())
> 	}

Reply:
good catch, thanks

Reply:
Actually setupLog does more stuff in the kubebuilder client for logging config. I'll keep it as it is for client codebase consistency.